core/src/test/java/org/apache/accumulo/core/data/TableIdTest.java

```diff
@@ -88,12 +88,13 @@ public void testCacheIncreasesAndDecreasesAfterGC() {
     assertEquals(tableString, table1.canonical());
 
     // create a bunch more and throw them away
-    for (int i = 0; i < 999; i++) {
-      TableId.of(new String("table" + i));
+    long preGCSize = 0;
+    int i = 0;
+    while ((preGCSize = TableId.cache.asMap().entrySet().stream().count()) < 100) {
```

Comment:
```suggestion
    while ((preGCSize = TableId.cache.asMap().entrySet().size()) < 100) {
```
There are also many other occurances in the test you could replace.

Reply:
This was a suggestion from my IDE but I don't think they are exactly the same...

Reply:
Yeah, the IDE is wrong in this case. See my other comment.